.github/workflows/pr-root-sync.yaml

```diff
@@ -21,17 +21,41 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: 'Set up Cloud SDK'
-        uses: 'google-github-actions/setup-gcloud@v2'
-        with:
-          version: '>= 472.0.0'
+      - name: Install kubeconform
```

Comment:
**Notes**:
- Run `kustomize build` first since it's fast
- Install `kubeconform` from the GitHub Release since I'm assuming it's faster than Docker